ingestion_tools/scripts/refactor_annotation_sources.py

```diff
@@ -0,0 +1,50 @@
+import yaml
+
+from common.yaml_refactor import get_files_to_refactor
+
+
+def main():
+    print(
+        "[WARNING]: Use with caution! This script overwrites existing config files, so make sure to still manually validate the changes. Comments, formatting, anchors, extends, etc. will be lost.",
+    )
+    if input("Do you want to continue? (y/n): ").lower() != "y":
+        return
+
+    # Get all files to validate
+    files_to_validate = get_files_to_refactor()
+
+    if not files_to_validate:
+        print("[WARNING]: No files to validate.")
+        return
+
+    for file in files_to_validate:
+        try:
+            with open(file, "r") as stream:
+                config_data = yaml.safe_load(stream)
+            file_changed = False
+            # If file has annotations, refactor sources
+            for entry in config_data["annotations"]:
+                new_sources = []
+                for source in entry["sources"]:
```

Comment:
nit: Validate that this is not already in the expected format, so we can even run this on files that have already been formatted without worrying about it. (Not a blocker)

Reply:
Done!